### PR TITLE
Adding word count program

### PR DESCRIPTION
**Pull request by Ihjass**
This is a program to print the count of words in the given sentence
word_count_output

### CODE REVIEW

File: src/strings/word_count.jl [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""
+wordcount(sentence)
+
+Program to find word count in the given sentence. The program will return count with the word.A full description of the algorithm can be found on [exercism](https://exercism.org/tracks/julia/exercises/word-count)
```
Comment (Panquesito7) [inline src/strings/word_count.jl:4]:
```suggestion
Program to find word count in the given sentence. The program will return count with the word.
A full description of the algorithm can be found on [exercism](https://exercism.org/tracks/julia/exercises/word-count)
```

File: src/strings/word_count.jl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+end
+
+```
+
+# References:
+(https://exercism.org/tracks/julia/exercises/word-count)
+
+```
+
+# Contributed by:- [Ihjass](https://github.com/Ihjass)
```
Comment (Panquesito7) [inline src/strings/word_count.jl:38]:
```suggestion
# Contributed by:- [Ihjass Thasbekha](https://github.com/Ihjass)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I think you need to add your program in the tests folder as seen in other files https://github.com/TheAlgorithms/Julia/tree/main/test

**Review by Panquesito7 — DISMISSED** [review]
Looks like all the other [**checks**](https://github.com/TheAlgorithms/Julia/pull/101/checks) are failing 😅 could you please fix that?
Let us know if you need any help here or in our [**Discord server**](https://discord.gg/c7MnfGFGa6). 😃

**Review by uncomfyhalomacro — DISMISSED** [review]
Seems you misplaced `input` instead of `sentence`

**Review by uncomfyhalomacro — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
LGTM! Thanks for helping, @uncomfyhalomacro! 🚀

**Pull request merged**